dev/custom-pages/research.html

```diff
@@ -0,0 +1,176 @@
+</p>
+<p>
+</div>
+<div class="col-sm-4 col-sm-offset-1">
+
+<a class="ucf-news-thumbnail external" href="http://www.ucf.edu/pegasus/going-public/" target="_blank"><img class="ucf-news-thumbnail-image" style="height: 240px;margin-top: 53px;" src="//wwwqa.cc.ucf.edu/wp-content/uploads/2017/01/going-public-1.png"></a>
+
+</div>
+</div>
+  </div>
+</section>
+<!-- Stats -->
+<section id="stats-section" data-section-link-title="Stats" class="callout callout-stats sections-menu-target homepage-section" style="background-image: url('//www.ucf.edu/wp-content/uploads/2016/12/MainPage-StatsBackgroundnoyellow-1.jpg'); background-position: 50% 50%;">
+  <div class="callout-stats-cover"></div>
+    <div class="container">
+      <div class="row">
+        <div class="col-md-6 col-sm-12">
+          <h2 class="h1 heading-slab-serif">University Research Impact</h2>
+          <p>UCF researchers attract more than $100 million annually in grants, gifts and sponsored awards. In addition to supporting innovative research, these financial resources help strengthen and diversify our local and state economy.</p>
+        </div>
+        <div class="col-md-6 col-sm-12">
+          <div class="stats-wrapper">
+            <div class="stats-item-row">
+              <div class="stats-item stats-border-top-left"><span class="num">$<span class="count-up" data-num="145.8">145.8</span></span> Million in Research Funding<br/><small>(2015-16)</small></div>
+              <div class="stats-item stats-border-top-right"><span class="num">#<span>19</span></span> Ranking Among Public Universities<br/><small>for Number of Patents, 2015</small></div>
+
+            <div class="stats-item-row">
+              <div class="stats-item stats-border-bottom-left"><span class="num"><span class="count-up" data-num="38">38</span></span> Licenses and Options<br/><small>(2015-16)</small></div>
+              <div class="stats-item stats-border-bottom-right"><span class="num"><span class="count-up" data-num="925">925</span></span> Total Patents Secured</div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+<!-- Clusters -->
+<section id="clusters-section" data-section-link-title="Clusters" class="callout callout-clusters">
+<div class="container">
+<h2 class="heading-underline">Multidisciplinary Research Clusters</h2>
+<p class="lead">UCF's Faculty Cluster Initiative leverages our diverse talent, facilities, location and industry partnerships to foster the development of strong, interdisciplinary teams focused on solving today's toughest scientific and societal challenges through teaching and research.
+</p>
+<div class="row">
+<div class="col-md-5 col-sm-6 col-xs-12 no-pad">
+<img class="img-responsive section-image" style="left: -15%;width: 125%;max-width: 125%;" src="//wwwqa.cc.ucf.edu/wp-content/uploads/2017/01/Research-Clusters-Graphic-1200x920.jpg">
```

Comment:
Inline styles should be applied via class.  Also make sure the negative positioning doesn't cause any horizontal scrollbars anywhere